content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -172,7 +172,7 @@ installing the `containerd.io` package can be found at
 {{% /tab %}}
 {{% tab name="Windows (PowerShell)" %}}
 
-Start a Powershell session, set `$Version` to the desired version (ex: `$Version="1.4.3"`), 
+Start a Powershell session, set `$Version` to the desired version (ex: `$Version="1.6.2"`), 
```

Comment:
I recommend using an example that is deliberately _not_ valid, and then helping the reader work out the right version.

Also, this page is likely to change: see https://github.com/kubernetes/website/pull/32738